site/en/docs/extensions/whatsnew/index.md

```diff
@@ -15,6 +15,12 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Chrome 117: Expanded kill-URL protections on extension API navigations {: #expanded-kill-url }
+
+<p class="color-secondary-text type--caption">Posted on <time>August 22, 2023</time></p>
+
+Previously navigation triggered from extension API calls for [`tabs.update()`](/docs/extensions/reference/tabs/#method-update), [`tabs.create()`](/docs/extensions/reference/tabs/#method-create), and [`windows.create()`](/docs/extensions/reference/windows/#method-create) emitted an error for some chrome:// URLs causing the browser to hang, or crash. Additionally, `tabs.update()` did not block JavaScript URLs. In 117, many errors caused by chrome:// URLs are eliminated and `tabs.update()` now blocks JavaScript URLs.
```

Comment:
Made a few tweaks, let me know what you think. We didn't fix anything crash related - we were just calling out that extensions aren't allowed to navigate to chrome://crash, which is a special debug URL that crashes the renderer for testing purposes. That was the case even before 117 (I realise this is confusing... 😅 ).

We could also replace all of the below with something much simpler if you prefer, like "JavaScript URLs can no longer be passed to `tabs.update`, and a number of additional chrome:// URLs have been blocked in `tabs.update`, `tabs.create` and `windows.create`.

```suggestion
Previously navigation triggered from extension API calls for [`tabs.update()`](/docs/extensions/reference/tabs/#method-update), [`tabs.create()`](/docs/extensions/reference/tabs/#method-create), and [`windows.create()`](/docs/extensions/reference/windows/#method-create) emitted an error for some chrome:// URLs. Additionally, calling `tabs.update()` with a JavaScript URL was forbidden. In 117, these protections on JavaScript URLs have been expanded to the `tabs.create()` method and a number of additional chrome:// URLs have been added to the list of forbidden URLs that applies to all of the previously mentioned methods.
```